### PR TITLE
Update GCP Private Link DNS Zone name to Response Policy Zone

### PR DESCRIPTION
**Pull request by fiquick**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-367.surge.sh

Updated pages:
- [Secure connections](https://neo4j-docs-aura-367.surge.sh/aura/platform/security/secure-connections)

**Comment by fiquick** [comment]
[Update GCP Private Link DNS Zone name to Response Policy Zone](https://trello.com/c/yze2NXRb/293-update-gcp-private-link-dns-zone-name-to-response-policy-zone)

File: modules/ROOT/pages/platform/security/secure-connections.adoc [hunk 1]
```diff
@@ -129,7 +129,7 @@ To continue accessing Browser and Bloom, you can configure a https://cloud.googl
 ====
 To access Bloom and Browser over a VPN, you must ensure that:
 
-* You have setup https://cloud.google.com/dns/docs/zones/manage-response-policies[GCP Cloud DNS], or an equivalent DNS service, inside of the VPC.
+* You have setup https://cloud.google.com/dns/docs/zones/manage-response-policies[GCP Response Policy Zone], or an equivalent DNS service, inside of the VPC.
```
Comment (AlexicaWright) [inline modules/ROOT/pages/platform/security/secure-connections.adoc:132]:
```suggestion
* You have set up link:https://cloud.google.com/dns/docs/zones/manage-response-policies[GCP Response Policy Zone], or an equivalent DNS service, inside of the VPC.
```

**Review by AlexicaWright — COMMENTED** [review]
Small suggestion on the format.

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-aura-367.surge.sh

Updated pages:
- [Secure connections](https://neo4j-docs-aura-367.surge.sh/aura/platform/security/secure-connections)

**Pull request merged**